src/main/scala/org/broadinstitute/dsde/workbench/sam/db/tables/PetServiceAccountTable.scala

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.dsde.workbench.sam.db.tables
+
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+import org.broadinstitute.dsde.workbench.model.{GoogleSubjectId, WorkbenchEmail, WorkbenchUserId}
+import org.broadinstitute.dsde.workbench.sam.db.SamTypeBinders
+import scalikejdbc._
+
+final case class PetServiceAccountRecord(userId: WorkbenchUserId,
```

Comment:
Is userId a primary key?

Reply:
It is a primary key here and in the user table. However, we are using the pre-existing `WorkbenchUserId` case class since it can be passed around easier rather than using a new case class like `UserIdPK`. The `userId` is also a bit different in that it is a string and not an auto-incrementing sequence like the other `DatabaseKey` extending classes that we are using in other tables.